composer.json

```diff
@@ -34,7 +34,6 @@
         "symfony/http-foundation": "^2.8 || ^3.2 || ^4.0",
         "symfony/http-kernel": "^2.8 || ^3.2 || ^4.0",
         "symfony/options-resolver": "^2.8 || ^3.2 || ^4.0",
-        "symfony/templating": "^2.8 || ^3.2 || ^4.0",
```

Comment:
Can you check if we can get rid of framework-bundle also? AFAIK it was used for the EngineInterface, not sure if we need it for anything else

Reply:
`symfony/framework-bundle` is used only in `Sonata\BlockBundle\Command\BaseCommand`:

```php
namespace Sonata\BlockBundle\Command;

use Sonata\BlockBundle\Block\BlockServiceManagerInterface;
use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;

abstract class BaseCommand extends ContainerAwareCommand { /* ... */ }
```

To remove dependency `extends ContainerAwareCommand` could be replaced with `implements \Symfony\Component\DependencyInjection\ContainerAwareInterface` here